workspaces/acr/package.json

```diff
@@ -52,6 +53,193 @@
     "typescript": "~5.3.0"
   },
   "resolutions": {
+    "@backstage/app-defaults": "1.6.5",
+    "@backstage/backend-app-api": "1.2.6",
+    "@backstage/backend-defaults": "0.12.0",
+    "@backstage/backend-dev-utils": "0.1.5",
+    "@backstage/backend-dynamic-feature-service": "0.7.3",
+    "@backstage/backend-openapi-utils": "0.6.0",
+    "@backstage/backend-plugin-api": "1.4.2",
+    "@backstage/backend-test-utils": "1.8.0",
+    "@backstage/catalog-client": "1.11.0",
+    "@backstage/catalog-model": "1.7.5",
+    "@backstage/cli": "0.34.1",
+    "@backstage/cli-common": "0.1.15",
+    "@backstage/cli-node": "0.2.14",
+    "@backstage/codemods": "0.1.52",
+    "@backstage/config": "1.3.3",
+    "@backstage/config-loader": "1.10.2",
+    "@backstage/core-app-api": "1.18.0",
+    "@backstage/core-compat-api": "0.5.1",
+    "@backstage/core-components": "0.17.5",
+    "@backstage/core-plugin-api": "1.10.9",
+    "@backstage/create-app": "0.7.3",
+    "@backstage/dev-utils": "1.1.13",
+    "@backstage/e2e-test-utils": "0.1.1",
+    "@backstage/errors": "1.2.7",
+    "@backstage/eslint-plugin": "0.1.11",
+    "@backstage/frontend-app-api": "0.12.0",
+    "@backstage/frontend-defaults": "0.3.0",
+    "@backstage/frontend-dynamic-feature-loader": "0.1.4",
+    "@backstage/frontend-plugin-api": "0.11.0",
+    "@backstage/frontend-test-utils": "0.3.5",
+    "@backstage/integration": "1.17.1",
+    "@backstage/integration-aws-node": "0.1.17",
+    "@backstage/integration-react": "1.2.9",
+    "@backstage/release-manifests": "0.0.13",
+    "@backstage/repo-tools": "0.15.1",
+    "@techdocs/cli": "1.9.7",
+    "@backstage/test-utils": "1.7.11",
+    "@backstage/theme": "0.6.8",
+    "@backstage/types": "1.2.1",
+    "@backstage/ui": "0.7.0",
+    "@backstage/version-bridge": "1.0.11",
+    "@backstage/plugin-api-docs": "0.12.10",
+    "@backstage/plugin-api-docs-module-protoc-gen-doc": "0.1.10",
+    "@backstage/plugin-app": "0.2.0",
+    "@backstage/plugin-app-backend": "0.5.5",
+    "@backstage/plugin-app-node": "0.1.36",
+    "@backstage/plugin-app-visualizer": "0.1.22",
+    "@backstage/plugin-auth-backend": "0.25.3",
```

Comment:
omg are all these dependencies used?

Reply:
Because we still want update to Backstage 1.42.x before we update to Backstage 1.43.0.

And unluckily Backstage 1.43.0 is already released and a lot of transitive dependencies uses then a "too new version" otherwise.

I know there is this yarn plugin but it doesn't work for me in the past. Maybe I missed something...?

I will remove all this resolutions in a follow up PR for Backstage 1.43.0 again. Do you see another option? :man_shrugging: